rego-templates/incident-jira.rego

```diff
@@ -18,6 +20,7 @@ result = msg {
     input.category,
 	input.severity_score,
     input.data,
-    input.response_policy_name
+    input.response_policy_name,
+    flat_array_with_default(input, "application_scope", "")
```

Comment:
it's going to be serialized as [\"scope1\", \"scope2\", \"scope3\"] which I think is not expected result

Reply:
Yes, I want to print all the applications scopes as an array